### PR TITLE
Qrda1 r5

### PR DESCRIPTION
**Pull request by dczulada**
This adds QRDA Category 1 R5 support.
Exports/Imports have been roundtrip tested with the current Cypress bundle (with eCQM for 2019 reporting).  Test patients were converented from QDM to HDS using the cqm-converter prior prior to export. 
 
**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] This PR is into the correct branch.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links back to this PR
- [ ] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass
- [ ] Code coverage has not gone down and all code touched or added is covered.
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here:
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here:
 
**Cypress Reviewer:**
 
Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code
 
**Bonnie Reviewer:**
 
Name: @mayerm94
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: templates/cat1/r5/_2.16.840.1.113883.10.20.24.3.38.cat1.erb [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-      <%== render(:partial=> 'result_value', :locals => {:values => entry.values}) %>
-    <% else %>
-      <value xsi:type="PQ" nullFlavor="UNK"/>
+      <%== render(:partial => 'results', :locals => {:entry => entry, :result_oids => result_oids, :value_set_oid => value_set_oid}) %>
+    <% end %>
+    <% if entry.components -%>
+      <%== render(:partial => 'components', :locals => {:entry => entry} ) %>
     <% end %>
-    <%== render(:partial => 'reason', :locals => {:entry => entry, :reason_oids=>field_oids["REASON"]}) %>
     </observation>
```
Comment (mayerm94) [inline templates/cat1/r5/_2.16.840.1.113883.10.20.24.3.38.cat1.erb]: There's an extra tab here

**Review by mayerm94 — DISMISSED** [review]
Reviewed all template changes. Confirmed all retired templates have been removed. Confirmed all introduced SHALL clauses have been added to the existing templates.

File: lib/health-data-standards/import/c32/patient_importer.rb [hunk 2]
```diff
@@ -141,9 +141,9 @@ def get_demographics(patient, doc)
           
           # parse race, ethnicity, and spoken language
           race_node = patient_element.at_xpath('cda:raceCode')
-          patient.race = { code: race_node['code'], code_set: 'CDC-RE' } if race_node
+          patient.race = { 'code' => race_node['code'], 'codeSystem' => 'CDC Race' } if race_node
```
Comment (mayerm94) [inline lib/health-data-standards/import/c32/patient_importer.rb]: VSAC lists the codesystem for races/ethnicities as 'CDCREC', not 'CDC Race'

**Review by laclark22 — DISMISSED** [review]
This looks good to me. I tested round trips for exports/imports in Cypress with the current bundle and things look good. Codes changes also look A okay to me as well.

**Review by c-monkey — APPROVED** [review]
HDS and Bonnie tests pass. Looks good.
